migrations/0002_migrations.rb

```diff
@@ -4,6 +4,7 @@
     create_table(:db_resources) do
       primary_key :id
       foreign_key :project_id, :projects
+      foreign_key :db_project_id, :db_resources_projects
```

Comment:
I don't think `node` or `DB` resources need to know they are in a `resource_projects` table
For example, If we are referencing the resource<->projects relationships I don't think we will look a the `db_resources` table for them